docs/spec/abci/abci.md

```diff
@@ -347,8 +347,10 @@ Commit are included in the header of the next block.
   - `Version (Version)`: Version of the blockchain and the application
   - `ChainID (string)`: ID of the blockchain
   - `Height (int64)`: Height of the block in the chain
-  - `Time (google.protobuf.Timestamp)`: Time of the block. It is the proposer's
-    local time when block was created.
+  - `Time (google.protobuf.Timestamp)`: Time of the previous block.
```

Comment:
Isn't this time of the block (not previous block)?

Reply:
No, we're using LastCommit to calculate median time https://github.com/tendermint/tendermint/blob/f965a4db15796dfca4e504d93eaa83760ced4af2/state/state.go#L149

Reply:
We should've renamed `Time` to `LastTime` or something. But now given we're switching to proposer's local time (basically reverting the changes), it does not worth it.

Reply:
We use LastCommit of block h to calculate time for block h+1. But it's still time of the current block (and shouldn't be renamed to LastTime).